charts/zot/templates/deployment.yaml

```diff
@@ -35,16 +35,50 @@ spec:
             - name: zot
               containerPort: 5000
               protocol: TCP
+          {{- if or .Values.mountConfig .Values.persistence }}
+          volumeMounts:
+          {{- if .Values.mountConfig }}
+            - mountPath: '/etc/zot'
+              name: {{ .Release.Name }}-secret
+          {{- end }}
+          {{- if and .Values.persistence }}
+            - mountPath: '/var/lib/registry'
+              name: {{ .Release.Name }}-volume
+          {{- end }}
+          {{- end }}
           livenessProbe:
             httpGet:
               path: /v2/
               port: 5000
+              {{- if .Values.authHeader }}
```

Comment:
Seems like the values.yaml have loss the description of `authHeader `.

Reply:
Yes, you are right, I have a comment on the actual `values.yaml` file I'm using, but not on the patch, the comment on my file is similar to this one, I'll add it to the patch:
```
# Authentication string for tests, it contains a `user:password` string encoded
# in base64. It is needed when `htpasswd` authentication is enabled and the
# default access does not provide read permission)
# The example value is from running `echo -n "foo:var" | base64`
#authHeader: "Zm9vOmJhcg=="
```

Reply:
Added on my branch